src/Lykke.Service.PayInternal/Models/TransferRequestMultiBijectiveModel.cs

```diff
@@ -0,0 +1,93 @@
+﻿using Lykke.Service.PayInternal.Core;
+using Lykke.Service.PayInternal.Core.Domain.Transfer;
+using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+
+namespace Lykke.Service.PayInternal.Models
+{
+    /// <summary>
+    /// The data to be obtained by TransferRequestController via http-request for multiple bijective transfer.
+    /// </summary>
+    public class TransferRequestMultiBijectiveModel : ITransferRequestModel
+    {
+        /// <summary>
+        /// ID of the Merchant who is responsible for money transfer.
+        /// </summary>
+        [Required]
+        public string MerchantId { get; set; }
+        /// <summary>
+        /// The side who pays the fee. If Merchant pays, then the Client receives the full sum
+        /// of money as it is pointed in request for the transfer, and the fee is additionaly 
+        /// paid from Merchant's wallets(s). Alternatively, the Client gets his (money - fee),
+        /// and the Merchant is not responsible for anything.
+        /// </summary>
+        [Required]
+        public TransferFeePayerEnum FeePayer { get; set; }
+        /// <summary>
+        /// The list of pair addresses for source-destination with the amount of transfer specified.
+        /// </summary>
+        [Required]
+        public List<BiAddressAmount> BiAddresses { get; set; }
+
+        /// <summary>
+        /// Indicates the first revealed validation error.
+        /// </summary>
+        /// <returns>The first encountered format mismatch or null if nothing was found.</returns>
+        /// <remarks>The mismatches that can be found here are:
+        /// * if one of bijective addresses has amount less than or equal to 0.</remarks>
+        public string CheckAmountsValidity()
```

Comment:
The validation result can't be message. Consider using enums or bool.

Reply:
If we introduce some enum here, we will have to bring additional messy logic to the http-controller level, for the controller itself will be responsible for enum-to-message convertation. Not sure it would be fine.

Reply:
It is the point of separation of concerns. Let's assume you have another consumer of this method but depending on validation result it needs to execute its business logic. You make it check the text message to define the validation result. Don't worry of putting the responsibility of enum-to-message conversion on controller. It's controllers responsibility to prepare the response.